nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/CONTRIBUTING.md

```diff
@@ -52,6 +53,76 @@ These tests are run both with the latest available version of `Nextflow` and als
 * Fix the bug, and bump version (X.Y.Z+1).
 * A PR should be made on `master` from patch to directly this particular bug.
 
-## Getting help
+## Getting Help
```

Comment:
I think the vast majority of nf-core docs only capitalise the first word of headings. So I'd prefer to change the others and leave this one as it is given the choice.